client/webserver/site/src/js/app.js

```diff
@@ -417,17 +417,18 @@ export default class Application {
    * updateExchangeRegistration updates the information for the exchange
    * registration payment
    */
-  updateExchangeRegistration (dexAddr, isPaid, confs) {
+  updateExchangeRegistration (dexAddr, isPaid, confs, asset) {
     const dex = this.exchanges[dexAddr]
 
     if (isPaid) {
       // setting the null value in the 'confs' field indicates that the fee
       // payment was completed
-      dex.confs = null
+      dex.pendingFee = null
       return
     }
 
-    dex.confs = confs
+    const symbol = this.assets[asset].symbol
+    dex.pendingFee = { confs, asset, symbol }
```

Comment:
NOTE: these JS changes with `pendingFee` in particular are not super robust yet.  You can get a JS error if you register via BTC with `dexcctl` while the page is opened.  But the current interactive registration workflow with the browser only is smooth and without error.